scripts/manage-db.py

```diff
@@ -31,40 +31,75 @@ def cleanup_releases(trans, nightly_age, dryrun=True):
 """ % nightly_age
     if dryrun:
         todelete = trans.execute("SELECT name FROM releases" + query).fetchall()
+        print "Releases rows to be deleted:"
         if todelete:
-            print "Releases rows to be deleted:"
             print "\n".join(itertools.chain(*todelete))
+        else:
+            print "  - None"
     else:
         trans.execute("DELETE releases FROM releases" + query)
 
 
 def cleanup_releases_history(trans, dryrun=True):
-    query = """
-WHERE name LIKE '%%%%latest'
-AND timestamp<1000*UNIX_TIMESTAMP(NOW()-INTERVAL 14 DAY);
-"""
+    num_to_delete = 100
+    find_records_latest = """
+        SELECT name, change_id
+        FROM releases_history
+        WHERE name LIKE '%%%%latest'
+          AND timestamp<1000*UNIX_TIMESTAMP(NOW()-INTERVAL 14 DAY)
+        ORDER BY change_id
```

Comment:
Is this ORDER BY there for a reason other than making sure we always delete from oldest -> newest? If it's expensive, we could forego it.